test/clojure-mode-font-lock-test.el

```diff
@@ -780,13 +780,13 @@ DESCRIPTION is the description of the spec."
      (2 4 font-lock-type-face)
      (5 5 nil)
      (6 9 font-lock-keyword-face)
-     (11 13 font-lock-function-name-face))
+     (11 13 nil))
 
     ("(clo/defrecord foo nil)"
      (2 4 font-lock-type-face)
      (5 5 nil)
      (6 14 font-lock-keyword-face)
-     (16 18 font-lock-function-name-face))
+     (16 18 nil))
```

Comment:
Seems to me this example needs to be updated as well to use `clojure.core`, as I doubt they would be a different ns for `defrecord`. Also we should probably make `defrecord` and `deftype` use `font-lock-type-face` for the name of the type.

Reply:
Replaced namespace with `clojure.core`.

> Also we should probably make defrecord and deftype use font-lock-type-face for the name of the type.
We already do, I didn't change that. I changed the test to check if this face is used

Reply:
Isn't 16-18 the name of the type `foo`? It seems it has no font-locking now.

Reply:
Ah, I see this wasn't working because of the wrong namespace. My bad.